wiki/Installation-guide.md

```diff
@@ -1,114 +1,84 @@
 * Install SteamVR, **launch it once** then close it.
 * Run `alvr_dashboard` or ALVR from your DE's application launcher.
 
-### Semi-Automatic Distrobox installation and guidance
-
-Notes:
-
-* This should be used in case you can't install or have issues with ALVR on your system. Do note that on Nvidia it uses host drivers and cuda nvenc, so if you're on older distribution like Debian and having older driver and cuda version, it will not work properly either way.
-
-* Guide also contains fixes, tweaks, additional software like desktop overlay to help you run with steamvr better and workaround it's issues.
-
-Installation:
-
-1. Download zip from https://github.com/alvr-org/ALVR-Distrobox-Linux-Guide, unpack it somewhere in your home directory (steam doesn't like long paths) or `git clone https://github.com/alvr-org/ALVR-Distrobox-Linux-Guide.git`.
-   
-   If you want to use nightly (potentially unstable, but fresh) builds, set `IS_NIGHTLY=0` variable in `env.sh` to `IS_NIGHTLY=1`.
-
-2. `cd ALVR-Distrobox-Linux-Guide`
-
-3. Carefully follow the [guide](ALVR-in-distrobox.md).
-
-4. Any issues related to this installer/various tweaks/bugs should be reported as issue [here](https://github.com/alvr-org/ALVR-Distrobox-Linux-Guide/issues)
-
-### Other
-
-#### AppImage
-
-You can get appimage for latest stable version from [here](https://github.com/alvr-org/ALVR/releases/latest).
-
-#### Flatpak
+### Flatpak
 
 For Flatpak users, refer to the instructions [here](https://github.com/alvr-org/ALVR/wiki/Flatpak)
 
-#### Portable tar.gz
+## Advanced usage
 
-* Install FFmpeg with VAAPI/NVENC + DRM + Vulkan + x264/x265 support. You can use this [ppa:savoury1/ffmpeg5](https://launchpad.net/~savoury1/+archive/ubuntu/ffmpeg5) under Ubuntu.
-* Install SteamVR, **launch it once** then close it.
-* Download `alvr_streamer_linux.tar.gz` from the release [download page](https://github.com/alvr-org/ALVR/releases/latest).
-* Run `bin/alvr_dashboard`
+### Use ALVR together with third-party drivers
 
-### Automatic Audio & Microphone Setup
+By default ALVR disables other SteamVR drivers before starting. Among these drivers there is [Driver4VR](https://www.driver4vr.com/) for full body tracking. ALVR disables these drivers to maximize compatibility with every PC setup. You can disable this behavior by manually registering the ALVR driver. Go to the `installation` tab and click on `Register ALVR driver`. The next time you launch ALVR you will be able to use the other drivers concurrently.
 
-* Must be on v20.5.0+
+### Launch ALVR together with SteamVR
 
-* Pipewire required
+You can skip the ALVR Dashboard and open ALVR automatically together with SteamVR. Open ALVR, go to the `Installation` tab and click on `Register ALVR driver`.
```

Comment:
It's important to mention that the registration should be done when SteamVR is not already running. Otherwise it will be unregistered on shutdown.